app/index.html

```diff
@@ -71,6 +71,8 @@
       </div>
     </div>
 
+    <img src="images/loginarrow.png" class="loginarrow visible-lg-block" alt="loginarrow" ng-if="!loggedIn">
```

Comment:
this is correct. Ng-if removes the element from the DOM, which is preferable. Ng-if requires a negation.